CHANGELOG.md

```diff
@@ -364,4 +364,14 @@ All notable changes to this project will be documented in this file. The format
  - Shared VIP service with mixed protocol doesn’t work if one of the protocol is SCTP.
 
 ### Known Issues
- - AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
\ No newline at end of file
+ - AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
+
+## AKO-1.10.2
+
+### Added
+ - AKO now claims support for K8 1.27.
+ - AKO chooses correct network reference for VIP subnet and Pool placement when there are duplicate networks, with same name, present on Avi Controller.
```

Comment:
Do you think we should include this under fixed since it is a bug fix?

Reply:
`Duplicate network being shown` is issue from NSX, VCenter and Avi controller side. AKO just added capability to detect right network from those duplicate network. I feel that if we add that as part of fixed issue, Customer will assume AKO had issue in choosing correct network.